pkg/cluster/controllers/storage/tenant.go

```diff
@@ -20,9 +20,13 @@ import (
 )
 
 var (
-	fsGroup    int64 = 10000
-	runAsGroup int64 = 10000
-	runAsUser  int64 = 10000
+	fsGroup            int64 = 10000
+	runAsGroup         int64 = 10000
+	runAsUser          int64 = 10000
+	ComponentName            = "cluster-minio"
+	ConfigImageKey           = "minio-docker-image"
+	MinioInitName            = "cluster-minio-init"
+	InitConfigImageKey       = "minio-docker-image-init"
```

Comment:
```suggestion
	InitConfigImageKey       = "minio-init-docker-image"
```

Reply:
@soulseen 

For new init image, you should also need to update the images.yaml under `manifests/cluster/patch`:

e.g:
```yaml
- name: HARBORCLUSTER_MINIO_INIT_DOCKER_IMAGE_2_2_1
  value: ghcr.io/goharbor/mc:tag
```